examples/mongodbatlas_privatelink_endpoint/aws/data-federation-online-archive/atlas-privatelink.tf

```diff
@@ -5,8 +5,10 @@ resource "mongodbatlas_privatelink_endpoint" "pe_east" {
 }
 
 resource "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive" "test" {
-  project_id    = var.project_id
-  endpoint_id   = aws_vpc_endpoint.vpce_east.id
-  provider_name = "AWS"
-  comment       = "Terraform Acceptance Test"
+  project_id                 = var.project_id
+  endpoint_id                = aws_vpc_endpoint.vpce_east.id
+  provider_name              = "AWS"
+  comment                    = "Terraform Acceptance Test"
```

Comment:
knitpit: i would change this comment as it's in the example folder and it's intended to be used by customers as a template, e.g.: "Example of data federation online archive"

Reply:
Updated in d62a3ae 👍